critiquebrainz/frontend/templates/place/entity.html

```diff
@@ -92,6 +93,11 @@ <h4 style="margin-bottom:0;">{{ _('Reviews') }}</h4>
 
   <div class="col-md-3">
     <h4>{{ _('Place information') }}</h4>
+      {% if avg_rating %}
+        <p>
```

Comment:
What is the `<p>` for here?

Reply:
It is needed to maintain the spacing format.

Reply:
You should use CSS for this.

Reply:
IMO, CSS isn't necessary here. Other elements in this column (in event and release_group) also use <p>, so we get perfectly formatted column without any use of CSS and by using mere <p>.

Reply:
There are plenty of places that are not implemented correctly. The goal is to make the code better, not to keep using bad practices. When other code has been written it wasn't reviewed properly.